### ISSUE TITLE
Componente `autocomplete` - Messaggi di stato in inglese

### ISSUE THREAD

**Issue by Fupete** [italia/bootstrap-italia#1139]
## Cosa

Dai test con lettori di schermo il componente **autocomplete restituisce messaggi di stato e hint in inglese**. 

Si segnalano ad esempio le due righe del `js` originale dove i default sono in inglese [51](https://github.com/alphagov/accessible-autocomplete/blob/79370c8c66f7496fb503bb1840cd590157f97232/src/autocomplete.js#L51) e [52](https://github.com/alphagov/accessible-autocomplete/blob/79370c8c66f7496fb503bb1840cd590157f97232/src/autocomplete.js#L52). 

È da capire come sostituire queste stringhe nella versione italiana usata in Bootstrap Italia. Potrebbe esserci la soluzione già nel readme originale, vedi [questo paragrafo ](https://github.com/alphagov/accessible-autocomplete?tab=readme-ov-file#tassistivehint-default---when-autocomplete-results-are-available-use-up-and-down-arrows-to-review-and-enter-to-select--touch-device-users-explore-by-touch-or-with-swipe-gestures) e [questo](https://github.com/alphagov/accessible-autocomplete?tab=readme-ov-file#tstatusnoresults-default---no-search-results) per quelle due stringhe in particolare.

È inoltre da verificare ulteriormente che non ci siano altri messaggi di stato (o di altro tipo) in inglese nel componente usato, si consiglia di verificare di aver tradotto tutte le stringhe usate nel componente UK originale. 

## Perché 

Comprensibilità. 

## Contesto

Verifiche di accessibilità sui componenti. 

Sarà da aggiornare lo status delle verifiche nella [board census ](https://github.com/orgs/italia/projects/32/views/1) alla risoluzione.

## Altro
Il bug è stato verificato su Bootstrap Italia, ma si immagina presente anche nei kit React e Angular.

**Comment by Fupete** [comment]
cc @frmascia11y @astagi @zetareticoli

**Comment by Fupete** [comment]
**Reminder:** @astagi da ricordarsi di aprire una issue su React per implementare la stessa soluzione, alla risoluzione di questa
